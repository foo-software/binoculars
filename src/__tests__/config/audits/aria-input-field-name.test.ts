import auditAriaInputFieldName from '../../../config/audits/aria-input-field-name';

describe('aria-input-field-name audit', () => {
  it('should match snapshot', () => {
    expect(auditAriaInputFieldName('en')).toMatchSnapshot();
  });
});
