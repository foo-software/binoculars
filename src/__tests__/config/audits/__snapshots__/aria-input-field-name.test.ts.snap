// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`aria-input-field-name audit should match snapshot 1`] = `
Object {
  "description": "When an input field doesn't have an accessible name, screen readers announce it with a generic name, making it unusable for users who rely on screen readers. [Learn more](https://web.dev/aria-input-field-name/).",
  "title": "ARIA input fields have accessible names",
}
`;
